src/v2/Apps/Categories/Components/GeneFamily.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Link, Spacer, Text, themeProps } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { GeneFamily_geneFamily } from "v2/__generated__/GeneFamily_geneFamily.graphql"
+import { sortBy } from "lodash"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+interface GeneFamilyProps {
+  geneFamily: GeneFamily_geneFamily
+}
+
+type Genes = GeneFamily_geneFamily["genes"]
+
+const alphabetizeGenes = (genes: Genes): Genes =>
+  sortBy(genes, gene => gene?.displayName || gene?.name)
+
+export const GeneFamily: React.FC<GeneFamilyProps> = props => {
+  const { geneFamily } = props
+  const { name, genes } = geneFamily
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
+
+  if (!genes) {
+    return null
+  }
+
+  const publishedGenes = genes.filter(g => !!g && g.isPublished)
+  const sortedGenes = alphabetizeGenes(publishedGenes)
+
+  return (
+    <Box id={`jump--${geneFamily.slug}`}>
+      <Text variant="xl">{name}</Text>
+      <Spacer mt={4} />
+      <ul style={{ columnCount: isMobile ? 1 : 3 }}>
```

Comment:
... can somehow get this wrapped into styled-system and then use responsive props? 

```tsx
import { columnCount } from 'styled-system' 
... 
const UL = styled.ul`
  ${columnCount};
`
...
<UL columnCount={[1, 3]}>
  ... 
</UL>
```

Reply:
☝️

Reply:
Turns out styled-system doesn't actually include `columnCount`, but I did realize that the `Masonry` component does exactly what we need here, so switched to that

Reply:
Ah yeah, doesn't provide it, but it helps us define it 😎 https://github.com/artsy/force/blob/d8a12dc1d291d2b1b9123209bd4e29ef91a09948/src/v2/Components/Masonry.tsx#L10-L13

Basically, any CSS prop can be made responsive like this, just by passing in a string. Best lib ever.